utils/src/test/java/org/nervos/ckb/crypto/HashTest.java

```diff
@@ -0,0 +1,91 @@
+package org.nervos.ckb.crypto;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.nervos.ckb.utils.Numeric.asByte;
+
+public class HashTest {
+
+    @Test
+    public void testSha3() {
+        byte[] input = new byte[] {
+                asByte(0x6, 0x8),
+                asByte(0x6, 0x5),
+                asByte(0x6, 0xc),
+                asByte(0x6, 0xc),
+                asByte(0x6, 0xf),
+                asByte(0x2, 0x0),
+                asByte(0x7, 0x7),
+                asByte(0x6, 0xf),
+                asByte(0x7, 0x2),
+                asByte(0x6, 0xc),
+                asByte(0x6, 0x4)
+        };
+
+        byte[] expected = new byte[]{
+                asByte(0x4, 0x7),
+                asByte(0x1, 0x7),
+                asByte(0x3, 0x2),
+                asByte(0x8, 0x5),
+                asByte(0xa, 0x8),
+                asByte(0xd, 0x7),
+                asByte(0x3, 0x4),
+                asByte(0x1, 0xe),
+                asByte(0x5, 0xe),
+                asByte(0x9, 0x7),
+                asByte(0x2, 0xf),
+                asByte(0xc, 0x6),
+                asByte(0x7, 0x7),
+                asByte(0x2, 0x8),
+                asByte(0x6, 0x3),
+                asByte(0x8, 0x4),
+                asByte(0xf, 0x8),
+                asByte(0x0, 0x2),
+                asByte(0xf, 0x8),
+                asByte(0xe, 0xf),
+                asByte(0x4, 0x2),
+                asByte(0xa, 0x5),
+                asByte(0xe, 0xc),
+                asByte(0x5, 0xf),
+                asByte(0x0, 0x3),
+                asByte(0xb, 0xb),
+                asByte(0xf, 0xa),
+                asByte(0x2, 0x5),
+                asByte(0x4, 0xc),
+                asByte(0xb, 0x0),
+                asByte(0x1, 0xf),
+                asByte(0xa, 0xd)
+        };
+
+        byte[] result = Hash.sha3(input);
+        assertThat(result, is(expected));
+    }
+
+    @Test
+    public void testSha3HashHex() {
```

Comment:
OK this is Keccak. We need SHA3's default (standard) algorithm, which hashes empty string to `a7ffc6f8bf1ed76651c14756a061d662f580ff4de43b49fa82d80a4b80f8434a`.

Reply:
OK, I will use standard SHA3 later